modal/app.py

```diff
@@ -658,6 +658,7 @@ def function(
         i6pn: Optional[bool] = None,  # Whether to enable IPv6 container networking within the region.
         # The next group of parameters are deprecated; do not use in any new code
         interactive: bool = False,  # Deprecated: use the `modal.interact()` hook instead
+        untrusted: bool = False,  # Whether the function is untrusted (e.g. user-provided code)
```

Comment:
Why is this going into the "deprecated" section?

Reply:
Nope - I just didn't look 2 lines above. Fixed.